dll/win32/cryptui/main.c

```diff
@@ -7376,15 +7373,13 @@ static LRESULT CALLBACK select_cert_dlg_proc(HWND hwnd, UINT msg, WPARAM wp, LPA
             }
             *data->cert = CertDuplicateCertificateContext(cert);
             free_certs(GetDlgItem(hwnd, IDC_SELECT_CERTS));
-            ImageList_Destroy(data->imageList);
             HeapFree(GetProcessHeap(), 0, data);
             EndDialog(hwnd, IDOK);
             break;
         }
         case IDCANCEL:
             data = (struct SelectCertData *)GetWindowLongPtrW(hwnd, DWLP_USER);
             free_certs(GetDlgItem(hwnd, IDC_SELECT_CERTS));
-            ImageList_Destroy(data->imageList);
             HeapFree(GetProcessHeap(), 0, data);
             EndDialog(hwnd, IDCANCEL);
```

Comment:
Very same comment as the one from above.

Reply:
I fully agree. Should I do it, surrounded by #ifdef __REACTOS__ ... #endif & #ifndef __REACTOS__ ... #endif ? Or do you prefer that i simply raise a WineHQ ticket and leave it as is ?

Reply:
See answer above.